src/daml/Daml/Finance/App/BackToBack/Service.daml

```diff
@@ -36,9 +35,9 @@ template Service
       with
         id : Id
         description : Text
-        quantity : Instrument.Q
-        customerAccount : Account.K
-        providerAccount : Account.K
+        quantity : InstrumentQuantity
```

Comment:
Agree with these changes. Should you also remove the type aliases `Q`, `K` from instrument? Can't see that file touched in this PR.

Reply:
We should, but unfortunately those type aliases are currently used by the getting started tutorial, which is shipped as part of the Daml SDK.

I believe removing those type aliases would constitute a breaking change to an interface package. However, we have no issue tracking this item, let me raise one.